pkg/plugins/resources/hcl/spec.go

```diff
@@ -0,0 +1,54 @@
+package hcl
+
+import (
+	"errors"
+
+	"github.com/sirupsen/logrus"
+)
+
+type Spec struct {
+	// [s][c][t] File specifies the hcl file to manipulate
```

Comment:
Comments here, are used to generated the documentation available both your IDE (jsonschema) and www.updatecli.io

I recently started improving it for other plugins, you can have a look to the yaml one https://github.com/updatecli/updatecli/blob/6da735541b50d74c6452832356cc9e7fc753fed2/pkg/plugins/resources/yaml/main.go#L17